src/legacy/state/swap/hooks.tsx

```diff
@@ -283,6 +283,7 @@ export function useDerivedSwapInfo(): DerivedSwapInfo {
     [inputCurrency, outputCurrency]
   )
 
+  // TODO: be careful! For native tokens we use symbol instead of address
```

Comment:
is this a TODO or just a comment?

Reply:
just nitpicking, but TODO in principle is something that would require an action, but there's no action here (to be fair,  unfortunately we are not actively cleaning up so also not biggy)